react-frontend/src/components/createRecognitionComponent.js

```diff
@@ -0,0 +1,218 @@
+import React, { useState, useReducer, useEffect } from "react";
+import { Form, Button, Col, Row, Card, InputGroup } from "react-bootstrap";
+import reducer from "components/reducers/createRecognitionReducer";
+import { setDetails } from "components/actions/createRecognitionAction";
+import { string, object } from "yup";
+import PropTypes from "prop-types";
+
+const initialState = {
+  user: "",
+  coreValue: "",
+  description: "",
+};
+
+const CreateRecognition = (props) => {
+  const {
+    userPlaceholder, // eslint-disable-line react/prop-types
+    users, // eslint-disable-line react/prop-types
+    coreValue, // eslint-disable-line react/prop-types
+    textareaPlaceholder, // eslint-disable-line react/prop-types
+    coreValuePlaceholder, // eslint-disable-line react/prop-types
+  } = props; // eslint-disable-line react/prop-types
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const [isLoading, setIsLoading] = useState(false);
+  const [validated, setValidated] = useState(false);
+  const [labelData, setLabelData] = useState("");
+
+  useEffect(() => {
+    console.log("Loading the page  " + isLoading); // eslint-disable-line no-console
```

Comment:
@jitendrasbunde , @mayuriardad 

[React Development Guidelines](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines) not met.